OracleWebLogic/samples/12213-webtier-apache/build_internal.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+#
+# Copyright (c) 2016-2018 Oracle and/or its affiliates. All rights reserved.
+#
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+#
+docker build --build-arg http_proxy=http://www-proxy.us.oracle.com:80 \
```

Comment:
This entire file needs to be removed. You cannot expose internal Oracle hosts publicly.